### ISSUE TITLE
Provide an npm package

### ISSUE THREAD

**Issue by arty-name** [mapcode-foundation/mapcode-js#7]
It would be nice to be able to use this module in Node.js. It's simplest to do when the code is provided as npm package. The current code seems to be difficult to use in Node.js.

**Comment by rijnb** [comment]
Sorry for the delay. Yes, good point.

**Comment by thany** [comment]
To be frank (and late 😃) the code is difficult to use in a browser as well.
It's introducing a million global variables, and the script could be written probably in half the amount of code at twice the speed if it would be proper modern ES6+ code.

**Comment by rijnb** [comment]
The speed seems hardly an issue. But indeed the code structure can be much improved. We would certainly welcome pull requests to achieve that. ;-)

**Comment by thany** [comment]
Soon™ I will be needing this component, and since I have unlimited time for my project (it's a hobby thing) and at work my coworkers and I require each other to write sensible speedy beautiful code, I shall be taking this on if it hasn't already by then.

But I don't really know what Soon™ means atm 😄

**Comment by willfarrell** [comment]
Yes, I would even break it up so the core functionality could be loaded without regional results returning.

Something:
- `@mapcode/core` aka international
- `@mapcode/region`
- `@mapcode/precision`

I haven't looked through the code to see if this makes sense yet.

@thany how's Soon™ going ^_^ ? I might jump on this.

**Edit:** Having looked through the code now, maybe baby steps and just get it on the npm. Iterate from there.

**Comment by thany** [comment]
@willfarrell I'm building my application first, before applying bells and whistles. I'm using a framework I've never used before so it's a learning experience, so it might take a good long while.

So feel free to jump on it, as you put it 😀
